CRM/Core/BAO/UserJob.php

```diff
@@ -146,7 +146,7 @@ public static function getStatuses(): array {
    * @return array
    */
   public static function getTypes(): array {
-    $types = Civi::cache()->get('UserJobTypes');
+    $types = Civi::cache('metadata')->get('UserJobTypes');
```

Comment:
I only thought this was relevant - but we should use the `metadata` cache here for consistency